Adding bloom command meta data, bloom group and bloom data type documentaion (#233)
valkey-io/valkey-doc: commands/bf.info.md

Returns information about a bloom filter

## Info Fields

* CAPACITY - Returns the number of unique items that would need to be added before scaling would happen

> * SIZE - Returns the number of bytes allocated

Reviewer: Are the info fields really in uppercase? In `INFO` and `CLUSTER INFO` the field names are in lowercase.

> * SIZE - Returns the number of bytes allocated

Author: You can specify a specific field using the uppercase or lowercase. Both options are accepted by the bf.info command

> * SIZE - Returns the number of bytes allocated

Reviewer: OK, so the argument is usually documented in uppercase, but the field name in the reply is in lowercase?

Is this list simultaneously documenting the arguments for filtering and the fields in the reply? It's slightly confusing. Maybe add a sentence to clarify this.

> * SIZE - Returns the number of bytes allocated

Author: Yes was being used to explain what fields mean with more context. As well as show how to get a specific field in case you don't want to get the full info reply. I will check and see if theres a better way of showing both of these things

> * FILTERS - Returns the number of filters in the specified key

Reviewer: I don't know. Let's use the name "bloom filter" for the object the user most often thinks about as a bloom filter. The name of some less important thing can use a less catchy name.

If users think of the top-level object as a single probabilistic filter, then I think the top-level object should be called bloom filter.

Are the multiple filters just an internal mechanism to make the filter scale, somehow like the dict uses two hashtables during incremental rehashing? If yes, then we can just call them internal filters, internal filter objects or something like that.

* ITEMS - Returns the number of unique items that have been added the the bloom filter
* ERROR - Returns the false positive rate for the bloom filter

> * EXPANSION - Returns the expansion rate

Author: Yes looks like it is nil for a non scaling filter
```
11) Expansion rate
12) (nil)
```

> * MAXSCALEDCAPACITY - Returns the maximum capacity that can be reached before an error occurs

Author: I couldn't see a way to link to the sub section in the topic file but I linked it to the topic file. If this link isn't useful if not to the specific section I can remove it

> Returns usage information and properties of a specific bloom filter

Reviewer 2: ```suggestion
Returns usage information and properties of a specific bloom filter.
```

## Info Fields

* CAPACITY - The number of unique items that would need to be added before a scale out occurs or (non scaling) before it rejects addition of unique items. 
* SIZE - The number of bytes allocated by this bloom filter.

> * FILTERS - Returns the number of filters in the specified key

Reviewer 2: ```suggestion
* FILTERS - Returns the number of sub filters contained within the bloom filter.
```

* ITEMS - The number of unique items that have been added to the bloom filter
* ERROR - The false positive rate of the bloom filter
* EXPANSION - The expansion rate of the bloom filter. Non scaling filters will have an expansion rate of nil.
* TIGHTENING - The tightening ratio of the bloom filter
* MAXSCALEDCAPACITY - The [maximum capacity](../topics/bloomfilters.md) that a scalable bloom filter can be expand to and reach before a subsequent scale out will fail.

> For non-scaling filters, the TIGHTENING and MAXSCALEDCAPACITY fields are not applicable and will not be returned, as they don't provide relevant functionality. When no optional fields are specified, the system will return all available fields for the given filter type.

Reviewer 2: ```suggestion
For non-scaling filters, the TIGHTENING and MAXSCALEDCAPACITY fields are not applicable and will not be returned.
When no optional fields are specified, all available fields for the given filter type are returned.
```
